defaults/main.yml

```diff
@@ -9,4 +9,6 @@ geoip_packages:
 geoip_databases: GeoLite2-Country
 geoip_user: "{{ ansible_ssh_user }}"
 geoip_group: "{{ ansible_ssh_user }}"
-geoip_config_location: "/home/{{ geoip_user }}/GeoIp.conf"
\ No newline at end of file
+geoip_config_location: "/home/{{ geoip_user }}/GeoIp.conf"
+geoip_account:
```

Comment:
I believe Ansible still would treat these as an undefined value. Can we set the defaults to the example values we are removing from the configuration file?